src/LibraryManager.Vsix/Json/Completion/LibraryIdCompletionEntry.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Imaging.Interop;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.Web.Editor.Completion;
+using Microsoft.VisualStudio.Text;
+using System.Diagnostics;
+using Microsoft.Web.LibraryManager.Providers.Unpkg;
+
+namespace Microsoft.Web.LibraryManager.Vsix
+{
+    internal class LibraryIdCompletionEntry : SimpleCompletionEntry
```

Comment:
>LibraryId [](start = 19, length = 9)

I think this should be a VersionCompletionEntry, not LibraryId

Reply:
As discussed, it will only handle the version entry.

---
In reply to: [198665923](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198665923) [](ancestors = 198665923)